test/services/validator.ts

```diff
@@ -0,0 +1,33 @@
+import Validator from 'tp-node/services/validator';
+import userSchema from 'tp-node/schemas/users';
+import {expect} from 'chai';
+import {ValidatorError} from 'tp-node/interfaces/validator';
+
+describe('Service validator', (): void => {
+    const validator = new Validator();
+
+    describe('validate()', (): void => {
+        it('should validate without error', (): void => {
+            const user = {
+                firstname: 'john',
+                lastname: 'doe',
+            };
+            validator.validate(user, userSchema, (err: ValidatorError | null, val: object | null): void => {
+                expect(err).to.be.null;
+                expect(val).to.deep.equal(user);
+            });
+        });
+
+        it('should validate with error', (): void => {
+            const user = {
+                firstname: 'john',
+            };
+            validator.validate(user, userSchema, (err: ValidatorError | null): void => {
+                expect(err).to.be.not.null;
+                if (err) {
```

Comment:
Ton assertion juste au dessus va fail avant, ton if sert pas

Reply:
![image](https://user-images.githubusercontent.com/6897865/59602380-dd4aaa00-9106-11e9-8916-e8ee79e64112.png)

Reply:
Dans ce cas:
`expect(err && err.message).to.be.string;`

Pour moi c'est une des raisons pour lesquelles il faut pas typer les tests